lib/Factory/ResolutionFactory.php

```diff
@@ -225,6 +225,13 @@ public function query($sortOrder = null, $filterBy = [])
                 $body .= ' AND intended_width > intended_height ';
             }
         }
+        
+        // for Layout Background form, always return Layout current resolution
+        // even if it is disabled
+        if ($parsedFilter->getInt('withCurrent') !== null) {
```

Comment:
I think this might be problematic as it will negate and other clause filtered. Perhaps it would be better if this filter just disabled the `enabled` one?

Reply:
The only place withCurrent is used is for the layout background form, which only uses that and enabled flag (now anyway).

In general though, it makes sense to move it inside the enabled condition to avoid potential issues.